src/app/components/select-field/index.html

```diff
@@ -164,33 +164,34 @@
   <sky-demo-page-properties sectionHeading="SkySelectFieldPickerContext properties">
     <sky-demo-page-property
       propertyName="data"
-      isOptional="true"
+      isOptional="false"
```

Comment:
This property is set by the select field itself and passed to the custom picker, as opposed to a field the consumer of the component sets. Therefore a "required" property doesn't really make sense on any of these properties.

Reply:
OK. I'll change it back. I just saw in https://github.com/blackbaud/skyux-select-field/blob/master/src/app/public/modules/select-field/select-field-picker-context.ts that this was the only property without the question mark or a default, so I thought that meant it was required.